tests/foreman/ui/test_user.py

```diff
@@ -639,6 +659,26 @@ def test_positive_update_email(self):
             self.user.validate_user(username, 'email', new_email)
 
     @tier1
+    def test_positive_update_description(self):
+        """Update Description in User
+
+        :id: f08ee305-0e0b-4df0-82d9-d10edcfa66c0
+
+        :expectedresults: User is updated successfully
+
+        :CaseImportance: Critical
+        """
+        description = gen_string('alpha')
+        new_description = gen_string('alpha')
+        username = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_user(session, username=username, description=description)
+            self.user.update(username, description=new_description)
```

Comment:
Yep, as @ldjebran said, use subtest & valid_data_list same way as in create test (you may take a look at similar update tests)